ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -80,6 +80,21 @@ interface {{ port_channel_interface }}
 {%         if port_channel_interfaces[port_channel_interface].rt is arista.avd.defined %}
       route-target import {{ port_channel_interfaces[port_channel_interface].rt }}
 {%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].redundancy is arista.avd.defined %}
+      redundancy {{ port_channel_interfaces[port_channel_interface].redundancy }}
+{%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].designated_forwarder_election is arista.avd.defined and port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined %}
+{%             set designated_forwarder_cli = "designated-forwarder election algorithm " %}
+{%             if port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined("modulus") %}
```

Comment:
Normally we don't make such hard tests if we just output the value. I suggest that you do the `if` to check for `preference` and then do an `else` and just output the value.

Reply:
okay, i'll remove the check for `modulus` and just check for `preference` and `dont-preempt`